config/routes.rb

```diff
@@ -6,11 +6,9 @@
 Rails.application.routes.draw do
 
   constraints PortConstraint.new(PrometheusExporterPort.call) do
-    require 'sidekiq/prometheus/exporter'
     require 'yabeda/prometheus/mmap'
 
-    mount Sidekiq::Prometheus::Exporter, at: '/metrics'
-    mount Yabeda::Prometheus::Exporter, at: '/yabeda-metrics'
+    mount Yabeda::Prometheus::Exporter, at: '/metrics'
```

Comment:
I wonder whether we should keep the old endpoint as well, as a means of deprecation for on-prem/RHOAM customers for a smoother migration?

I mean the data would be the same reduced one, just not cut off the old endpoint.